packages/core/bootstrap/schemas/env.json

```diff
@@ -29,6 +29,15 @@
     "API_VERBOSE": {
       "type": "boolean"
     },
+    "SERVER_RATE_LIMIT_MAX": {
+      "type": "string"
+    },
+    "SERVER_SLOW_DOWN_AFTER_FACTOR": {
+      "type": "string"
+    },
+    "SERVER_SLOW_DOWN_DELAY_MS": {
+      "type": "string"
```

Comment:
Should all of these be strings or int?

Reply:
Currently we don't type check them. When parsed from the environment they are always strings, so for that reason and consistency I set them as such.

I could see the argument for saying int here, though `SERVER_SLOW_DOWN_AFTER_FACTOR` is a float.